custom_platforms.js

```diff
@@ -320,7 +353,12 @@ module.exports = {
                 "\\$content",
                 ",content(,|$)",
                 "$webrtc",
-                "#\\$#@media "
+                "#\\$#@media ",
+                ",domain=\/",
```

Comment:
It looks like that something like this:
```adblock
test$xmlhttprequest,domain=/example\..*/
```
is already supported by uBlock Origin.
```suggestion
```